### PR TITLE
Ft enable social login #147322191

### PR DESCRIPTION
**Pull request by daisymacharia**
#### What does this PR do?
Enable social login.

#### Description of Task to be completed?
The user will be able to login using face book, google or twitter.

#### How should this be manually tested?
The user could try manually login in using their social media account.

#### Screenshots (if appropriate)
screen shot 2017-07-05 at 18 17 17
screen shot 2017-07-05 at 18 20 31

### CODE REVIEW

File: requirements_devel.txt [hunk 1]
```diff
@@ -7,5 +7,72 @@
 
 # Development packages
 
-django-debug-toolbar
+
 coverage
+alabaster==0.7.10
```
Comment (egichuri) [inline requirements_devel.txt]: There seems to be quite a number of packages not related to this task. please clean this up

File: wger/settings_global.py [hunk 2]
```diff
@@ -407,3 +395,18 @@
     'EMAIL_FROM': 'wger Workout Manager <wger@example.com>',
     'TWITTER': False
 }
+
+# facebook, twitter and google API keys
```
Comment (egichuri) [inline wger/settings_global.py:399]: Would be better to use env variables for this.

Comment (egichuri) [reply wger/settings_global.py:399]: For clarity... any sensitive information that our app needs should not be made public. While changing this please also use env variables for the secret key

**Review by egichuri — CHANGES_REQUESTED** [review]
Awesome work @daisymacharia 
Please work on the comments.

File: requirements_devel.txt (hunk 1 above)
Comment (egichuri) [reply requirements_devel.txt]: on second thought.. i'm merging a branch with the cleaned up requirements. you can update your branch with develop once that's done

Comment (daisymacharia) [reply requirements_devel.txt]: okay lemme work on that

File: wger/settings_global.py (hunk 2 above)
Comment (daisymacharia) [reply wger/settings_global.py:399]: okay i'll change that

**Comment by egichuri** [comment]
@daisymacharia just noticed a couple more places where we need to add the social login buttons. Let's pair on this on monday.

**Pull request merged**